styles.css

```diff
@@ -142,4 +146,64 @@
                 position: relative;
                 display: flex;
                 flex-direction: column;
-            }
\ No newline at end of file
+                align-items: center;
+            }
+
+        @media only screen and (max-width: 700px) {
+            .header{
+                width: auto;
+            }        
+              }
+        @media only screen and (max-width: 320px){
+            .header{
```

Comment:
wow, impressed with the responsiveness adjustments. 
Good job!
But they are valid for screens smaller than 320px which makes the design useless in 99.9% of smartphones ;)
adjust your mediaquery to include all devices starting from 320px to 575px

Reply:
thanks, done